de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/async/GenerateInOutFromShipmentSchedules.java

```diff
@@ -111,6 +111,7 @@
 
 	public static final String PARAM_IsUseQtyPicked = "IsUseQtyPicked";
 	public static final String PARAM_IsCompleteShipments = "IsCompleteShipments";
+	public static final String PARAM_IsShipmentDateToday = "IsShipmentDateToday";
```

Comment:
you could move those constants into that params-pojo I mentioned further up

Reply:
Done